SEP-0002-revised.md

```diff
@@ -0,0 +1,249 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Rationale
+
+This SEP describes the second major iteration of the governance for the SunPy Project.
+The objectives of the rewrite were to adapt the governance for the changes the project has seen since the first iteration of this document was written in 2014.
+This rewrite aims to detail a governance structure, which strives to be lightweight, while being designed to enable the project to grow, and facilitating more input from the wider SunPy community.
+
+# The SunPy Project
+
+The primary mission of the SunPy Project is *To develop and facilitate a high-quality, easy-to-use, community-led, free and open-source solar data analysis ecosystem based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* supporting a community of affiliated packages
+* managing and protect the SunPy brand and identity
+* promoting SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy Project's governance involves the following five components:
+
+* The Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project exists for the benefit of the whole Community: users, developers, and members of the wider solar physics and scientific software community.
+The Community is the ultimate authority in how the project is managed.
+This authority is exercised through the processes outlined in this document, primarily through the Steering Committee and the Project Members.
+Whenever possible, decisions shall be approved by consensus, considering input from the entire community, rather than taken unilaterally by Project Members or the Steering Committee.
+
+## Advisory Board
+The primary functions of the Advisory Board are:
+
+* to advise the Steering Committee on management of the project,
+* aid in finding funding for the project to ensure the core aims of the project can be met,
+* nominate a Ombudsperson,
+* ensure that the governance of the project is running as intended.
+
+The Advisory Board is self-electing, with members appointed by a vote of the existing Advisory Board.
+
+* **Membership**: The Advisory Board shall consist of at least 3 members and no more than 5 members.
+The Advisory Board selects how many members are needed.
+All members shall serve a maximum of a two-year term, however a term may be extended until a successor has been appointed.
```

Comment:
@dstansby and @mbobra propose this edit https://github.com/sunpy/sunpy-SEP/pull/62#discussion_r995849639

```suggestion
All members shall serve a maximum of a two-year term.
```